### DISCUSSION TITLE
Willow won't start

### DISCUSSION THREAD

**Discussion by skorokithakis** [toverainc/willow#245] [Q&A]
I flashed Willow from the web flasher, but I get this error when starting:

```
I (00:00:20.841) WILLOW/MAIN: Starting up! Please wait...
I (00:00:20.926) WILLOW/MAIN: SPIFFS mounted
I (00:00:20.926) WILLOW/CONFIG: opening /spiffs/user/config/willow.json
E (00:00:20.988) WILLOW/CONFIG: failed to open /spiffs/user/config/willow.json
W (00:00:25.068) PERIPH_WIFI: WiFi Event cb, Unhandle event_base:WIFI_EVENT, event_id:4
I (00:00:26.568) WILLOW/WAS: initializing WebSocket client (wss://willowserver.home.stavros.io/ws)
I (00:00:26.569) WILLOW/WAS: initializing WebSocket client (wss://willowserver.home.stavros.io/ws)

***ERROR*** A stack overflow in task websocket_task has been detected.


Backtrace: 0x4037a1be:0x3fcc4b50 0x40383fed:0x3fcc4b70 0x40387a1a:0x3fcc4b90 0x40385d68:0x3fcc4c00 0x403840a4:0x3fcc4c20 0x4038409a:0x4148e269 |<-CORRUPTED




ELF file SHA256: 5622c060075d1bb1

Rebooting...
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0xc (RTC_SW_CPU_RST),boot:0xa (SPI_FAST_FLASH_BOOT)
Saved PC:0x42146a92
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3808,len:0x170c
load:0x403c9700,len:0xc14
load:0x403cc700,len:0x33b4
entry 0x403c9968
```

It looks like the willow.json is missing, but why? Could it be because there's a space in my wifi SSID?

I also filed this as #244.

**Comment by tombloemker** [comment]
Yep. I'm stuck here as well. failed to open /spiffs.
Making WAS a ubuntu server seems to miss the first principles issue. It doesn't play well with Windows. This would have been better as a phone app. I'm sticking with the original, at least I got it to work.

**Comment by skorokithakis** [comment]
For me, the problem was that it couldn't properly connect to the server. Are you using a reverse proxy? This definitely needs a better error message.

**Reply by tombloemker** [reply]
Reverse proxy....   Last time I messed this one of those it was 2002. I recorded most of my problems in the Getting started [comment page](https://heywillow.io/quick-start-guide/). Yes, I could not connect to WAS, by host IP or WSL IP. Also, a solution that worked for Takore2002 didn't work for me. Stintel says I have a WIFI issue.

**Reply by skorokithakis** [reply]
Yeah, it was a connectivity issue for me too, in the end.
